src/components/mapgl-import/mapgl-import.component.ts

```diff
@@ -537,16 +531,14 @@ export class MapglImportComponent {
     } else {
       if (importedResult.geojson.features.length > 0) {
         this.dialogRef.componentInstance.isRunning = false;
-        this.mapComponent.map.getSource(this.SOURCE_NAME_POLYGON_IMPORTED).setData(importedResult.geojson);
-        this.mapComponent.map.getSource(this.SOURCE_NAME_POLYGON_LABEL).setData({
-          type: 'FeatureCollection',
-          features: importedResult.centroides
-        });
-
         if (this.fitResult) {
           this.mapComponent.map.fitBounds(extent(importedResult.geojson));
         }
         this.imported.next(importedResult.geojson.features);
+        if (this.mapComponent.drawData.features.length > 0) {
+          this.mapComponent.drawData.features.forEach(df => importedResult.geojson.features.push(df));
+        }
+        this.imported.next(importedResult.geojson.features);
```

Comment:
Why is there `this.imported.next(importedResult.geojson.features);` before and after the condition ?

Reply:
it is a mistake